### PR TITLE
Add a button to All News category below the Recent News on the main page for faster access to it

### PR DESCRIPTION
**Pull request by oleg-dubinskiy**
Add a button to All News category below the Recent News on the main page, for more easy access to it, same as I did it for Live Activity feed. Then it will not required to open the News -> All News at the top, what will easier, e. g. for newcomers, if they will want to immediately switch to all news.
Before:
news-before
After:
news-after

### CODE REVIEW

**Comment by ColinFinck** [comment]
Yes, that link is badly needed.
But wouldn't it be more appropriate to have it right-aligned and as a square button? (like we already have on e.g. https://pr71.web-content.reactos.org/news/)

grafik

**Comment by oleg-dubinskiy** [comment]
>But wouldn't it be more appropriate to have it right-aligned and as a square button?

Oh, indeed. That button probably will be better.
I guess that is made via HTML, not CSS.

**Comment by oleg-dubinskiy** [comment]
Fixed. Thanks @ColinFinck.
I made the button maximally similar to those "Newer" and "Older" buttons in "All News" category.
Although I used the same button style as on the 404 error page (with a bit circled corners), since I don't know how is named the button style with squared corners and where it is declared. Is the current button's style correct for it?
Here's a screenshot how it looks with my current changes:
изображение

**Comment by learn-more** [comment]
> Fixed. Thanks @ColinFinck.
> I made the button maximally similar to those "Newer" and "Older" buttons in "All News" category.
> Although I used the same button style as on the 404 error page (with a bit circled corners), since I don't know how is named the button style with squared corners and where it is declared. Is the current button's style correct for it?
> Here's a screenshot how it looks with my current changes:
> изображение

```
<p class="buttons">
    <a href="https://reactos.org/" class="btn btn-template-main"><i class="fa fa-home"></i> Go to Homepage</a>
</p>
```

**Comment by oleg-dubinskiy** [comment]
@learn-more so this button style is the one I need? How I can make the corners squared then?

**Comment by learn-more** [comment]
the `.btn` from `bootstrap.min.css` is the one that creates the rounded borders

**Comment by oleg-dubinskiy** [comment]
OK. Thanks.
After removing `btn` from `class`, the button has no rounded borders, but now, it is too small:
изображение
How I can make it bigger, same on that page? Need I use some predefined value or just can specify the borders height and width manually?
UPD: `style` attributes don't work for me as expected. They change the space around the button, but not the button size itself. :slightly_frowning_face:

**Comment by learn-more** [comment]
> OK. Thanks.
> After removing `btn` from `class`, the button has no rounded borders, but now, it is too small:
> изображение
> How I can make it bigger, same on that page? Need I use some predefined value or just can specify the borders height and width manually?
> UPD: `style` attributes don't work for me as expected. They change the space around the button, but not the button size itself. 🙁

This is the html from the page colin linked:

```
<ul class="pager">
    <li class="previous disabled"><a href="#">← Newer</a></li>
    <li class="next"><a href="https://pr71.web-content.reactos.org//news/page/2/">Older →</a></li>
</ul>
```

**Comment by Extravert-ir** [comment]
I think I see "Older posts" buttons more often than "More". So I'm more for the former

**Comment by oleg-dubinskiy** [comment]
Thanks again @learn-more. I just forgot to look at the code of that page.
Now the button looks completely correctly, as I wanted. Also I applied a suggestion from @Extravert-ir (use a better caption for the button):
изображение

File: themes/reactos/layouts/partials/recent_posts.html [hunk 1]
```diff
@@ -38,6 +38,11 @@ <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
 			</section>
 
 			{{ end }}
+
+		<ul class="pager">
+			<li class="next"><a href="https://pr71.web-content.reactos.org/news">Older posts →</a></li>
```
Comment (ColinFinck) [inline themes/reactos/layouts/partials/recent_posts.html:43]:
```suggestion
			<li class="next"><a href="/news">Older posts →</a></li>
```

Comment (oleg-dubinskiy) [reply themes/reactos/layouts/partials/recent_posts.html]: Fixed. Thanks @ColinFinck.
Now it looks even better and still works for me.

**Pull request merged**